.github/workflows/UpdateALGoOnOfficialBranches.yaml

```diff
@@ -0,0 +1,51 @@
+name: Update AL-Go On Official Branches
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "55 08 * * 3" # Run at 08:55 UTC on Wednesday
+
+defaults:
+  run:
+    shell: powershell
+
+jobs:
+  GetBranches:
+    name: Get Official Branches
+    runs-on: windows-latest
+    outputs:
+      UpdateBranches: ${{ steps.OfficialBranches.outputs.OfficialBranches }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get Official Branches
+        id: OfficialBranches
+        uses: ./.github/actions/GetOfficialBranches
+
+  UpdateALGo:
+    name: "[${{ matrix.branch }}] Update AL-Go"
+    permissions:
+      contents: write
```

Comment:
Does this need `contents: write` permission? Doesn't it just trigger "Update Al-Go System Files"?

Reply:
Good point!

Reply:
Removed unneeded permissions.